packages/apollo-server-hapi/src/hapiApollo.ts

```diff
@@ -54,7 +54,13 @@ const graphqlHapi: IPlugin = {
           });
 
           const response = h.response(gqlResponse);
-          response.type('application/json');
+          Object.keys(responseInit.headers).forEach(key =>
+            response.header(key, responseInit.headers[key]),
+          );
+          response.header(
```

Comment:
this wasn't needed before?

Reply:
The express variant contained this, so I thought that the hapi and koa variants should be at feature parity. The reason it's not inside of apollo-server-core is because of the cloudflare integration. We can move the calculation there and then decide how to do the Buffer length in a V8 environment later